pkg/namespacestore/namespacestore.go

```diff
@@ -265,11 +282,20 @@ func createCommon(cmd *cobra.Command, args []string, storeType nbv1.NSType, popu
 func RunCreateAWSS3(cmd *cobra.Command, args []string) {
 	createCommon(cmd, args, nbv1.NSStoreTypeAWSS3, func(namespaceStore *nbv1.NamespaceStore, secret *corev1.Secret) {
 		targetBucket := util.GetFlagStringOrPrompt(cmd, "target-bucket")
-		accessKey := util.GetFlagStringOrPromptPassword(cmd, "access-key")
-		secretKey := util.GetFlagStringOrPromptPassword(cmd, "secret-key")
 		region, _ := cmd.Flags().GetString("region")
-		secret.StringData["AWS_ACCESS_KEY_ID"] = accessKey
-		secret.StringData["AWS_SECRET_ACCESS_KEY"] = secretKey
+		secretName, _ := cmd.Flags().GetString("secret-name")
```

Comment:
Looks like this whole code can be in a function
The only difference is the keys, not the logic

Reply:
let's keep it as it is so the flags will be handled at the same place and won't need to pass 4 keys for each call.